packages/credential-governance/src/errors/chain-is-not-supported.ts

```diff
@@ -0,0 +1,5 @@
+export class ChainIsNotSupported extends Error {
```

Comment:
@JGiter Can we not put all the error classes in a single file? Is there any specific reason put them separate?

Reply:
I put them separately in order to not increase file size while adding new errors